superset/viz.py

```diff
@@ -840,16 +840,11 @@ def to_series(self, df, classed='', title_suffix=''):
             ys = series[name]
             if df[name].dtype.kind not in "biufc":
                 continue
-            if isinstance(name, string_types):
-                series_title = name
-            else:
-                name = ["{}".format(s) for s in name]
-                if len(self.form_data.get('metrics')) > 1:
```

Comment:
Users reported an issue related to removing this logic. In the case where there's a single metric and a one or many dimension, the metric name would not be reported prior to this change, which is the desired behavior. Where before using metric `count` with `gender` dimension would give ['female', 'male', 'other'] in both the legend and tooltip, now it says  ['count, female', 'count, male', 'count, other']

Reply:
Please don't report this to me if you don't want me to fix it. Just wasted 30min on context switching and pushing a branch only to find your fix...